app/views/home_page/show.html.slim

```diff
@@ -0,0 +1,7 @@
+div.jumbotron
+  h1.display-3 First page of the app
+  p.lead In the end it's aimed to be personal-finance management app
+  hr.my-4
+  p For now it's just a simple page
+  p.lead
+    a class=['btn','btn-primary','btn-lg'] role="button" Learn more
```

Comment:
@sinpai in real practice class names would be mentioned like that:
```slim
a.btn.btn-primary.btn-lg
```

instead of 
```slim
a class=['btn','btn-primary','btn-lg']
```